.changeset/wet-flies-joke.md

```diff
@@ -0,0 +1,8 @@
+---
+"@ima/plugin-cli": minor
+---
+
+ - `typescriptDeclarationsPlugin` now accepts optional `tsConfigPath`, which allows you to specify a custom path to your `tsconfig.json` file.
+ - `typescriptDeclarationsPlugin` now by default tries to resolve `tsconfig.build.json` before `tsconfig.json`, if it exists.
+ - `typescriptDeclarationsPlugin` now fails when types are not correctly generated, instead of silently ignoring the error. This can be disabled by setting `failOnTypeErrors` to `false`.
```

Comment:
This seems like a breaking change. Maybe we should set default value to `true` and we can change the default value with IMA 20?

Reply:
You're right, I forgot to change this, it is now disabled by default (to not produce breaking changes)